setup.cfg

```diff
@@ -40,5 +40,5 @@ asyncio_mode = auto
 
 [pydocstyle]
 inherit = false
-ignore = D200,D202,D213,D203,D205,D300,D400,D401,D405,D406,D407,D415
+ignore = D213,D203  # D203 and D211 are in conflict, D212 and D213 are in conflict
```

Comment:
Thanks for testing! For more info: https://github.com/PyCQA/pydocstyle/issues/475#issuecomment-627520864